src/MitoCount.jsx

```diff
@@ -404,6 +404,12 @@ function MitoCount(props) {
   const handleHelpClose = () => { setHelpOpen(false); };
 
   const handleKeyPress = (event) => {
+    // Ignore keyboard shortcuts when a Neuroglancer text input has focus.
+    const focus = document.activeElement;
+    if (focus && focus.type && focus.type.includes('text')) {
```

Comment:
Same suggestion as code in src/FocusedProofreading.jsx. Should we extract this to a common function in src/utils.js